tensorflow/core/kernels/dml_pad_op.cc

```diff
@@ -49,6 +49,7 @@ struct SimplePad {
 template <typename Tpadding>
 absl::optional<SimplePad> SimplifyPad(const TensorShape& input_shape,
                                       const Tensor& paddings_tensor,
+                                      bool constant_pad,
                                       size_t min_output_size = 4,
                                       size_t max_output_size = 5) {
```

Comment:
Can we increase this to 8 now?

Reply:
Especially since the error message was updated to 8D, but this constant hasn't.